### ISSUE TITLE
SCT fails to retrieve logs from AWS node when initializing cluster

### ISSUE THREAD

**Issue by kbr-** [scylladb/scylla-cluster-tests#2252]
```
< t:2020-06-12 15:17:37,224 f:cluster.py      l:1995 c:sdcm.cluster_aws     p:INFO  > Node bootstrap-kbraun-db-node-ca715558-1 [3.250.144.45 | 10.0.186.54] (seed: False): Found ScyllaDB version: 666.development
< t:2020-06-12 15:22:37,522 f:wait.py         l:64   c:sdcm.wait            p:ERROR > Wait for: wait for db logs: timeout - 300 seconds - expired
< t:2020-06-12 15:22:37,523 f:wait.py         l:68   c:sdcm.wait            p:ERROR > last error: Exception('No db log from node [Node bootstrap-kbraun-db-node-ca715558-1 [3.250.144.45 | 10.0.186.54] (seed: True)] ')
```

this happened on one of my runs of the test described in https://github.com/scylladb/scylla-cluster-tests/issues/2243. But this doesn't seem to be test specific: the test failed in the `setUp` phase.

I manually checked by SSHing into the node that the node has successfully started:
```
Jun 12 13:16:29 bootstrap-kbraun-db-node-ca715558-1 scylla[2039]: Scylla version 666.development-0.20200610.65d3e3992 with build-id e42da3a7b88f607f972cc487f08ce35faaa60fe5 starting ...
...
Jun 12 13:16:55 bootstrap-kbraun-db-node-ca715558-1 scylla[2039]:  [shard 0] init - Scylla version 666.development-0.20200610.65d3e3992 initialization completed.
```

[bootstraplog.zip](https://github.com/scylladb/scylla-cluster-tests/files/4771070/bootstraplog.zip)

**Comment by kbr-** [comment]
Actually it's not flakiness, I encountered this after updating SCT to master and now keep reproducing it.
It looks like master is broken.

This seems relevant:
```
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  > 2020-06-12 15:49:48.482: (ThreadFailedEvent Severity.ERROR): message='__class__'                                                                                                                                                        
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  > Traceback (most recent call last):                                       
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  >   File "/home/kbraun/dev/scylla-cluster-tests/sdcm/sct_events.py", line 1049, in wrapper                                         
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  >     result = func(*args, **kwargs)                                       
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  >   File "/home/kbraun/dev/scylla-cluster-tests/sdcm/utils/remote_logger.py", line 70, in _journal_thread                                                                                                                                 
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  >     self._remoter = self._remoter_params.pop('__class__')(**self._remoter_params)                                                
< t:2020-06-12 15:49:48,512 f:sct_events.py   l:920  c:sdcm.sct_events      p:INFO  > KeyError: '__class__'                                                    
```

**Comment by kbr-** [comment]
After reverting https://github.com/scylladb/scylla-cluster-tests/commit/b82d2bac19fc75104556488a3641d99e98fa7144 it works.

**Comment by ShlomiBalalis** [comment]
The same issue happens constantly to artifact tests. Test ids:
centos 7 - 66a26044-f300-4a27-8db0-e60362f6e95d
debain9 - 792af4ef-9700-4cd8-9fad-837d1cb7274c
debian10 - 1fdf3d33-ed01-4b2f-9210-9096aca45c53
ubuntu16 - b22009da-d1fd-4a71-85fd-5adef2302068
ubuntu18 - 89074188-e06f-46ef-8a4c-063ce48cc1b1
oel76 - ffc2c838-5cf4-49ec-a8f2-3b65b41097af

**Comment by roydahan** [comment]
@dkropachev / @bentsi please take a look on this.

**Comment by bentsi** [comment]
fixed in #2250
